client/my-sites/plugins/controller.js

```diff
@@ -74,7 +75,8 @@ function renderSinglePlugin( context, siteUrl, isWpcomPlugin ) {
 			isWpcomPlugin: isWpcomPlugin,
 			onPluginRefresh: title => titleActions.setTitle( title )
 		} ),
-		document.getElementById( 'primary' )
+		document.getElementById( 'primary' ),
+		context.store
```

Comment:
do you mean passing the whole store vs passing just the needed subtree? If so, I haven't read anything about it, but I can understand your concerns. Maybe we should pass the subtrees that are going to be used by the component... But what happens when the component is going to use two different subtrees? should we pass the whole store, or two different params with different subtrees?